packages/@lightningjs/ui-components/src/components/CardContent/CardContentHorizontal.d.ts

```diff
@@ -16,8 +16,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import CardContent, { CardContentStyle } from './CardContent';
+import lng from '@lightningjs/core';
+import CardContent from './CardContent';
 
-export type CardContentHorizontalStyle = CardContentStyle;
+declare namespace CardContentHorizontal {
+  export type TemplateSpec = CardContent.TemplateSpec;
+}
+declare class CardContentHorizontal<
+  TemplateSpec extends CardContent.TemplateSpec = CardContent.TemplateSpec,
```

Comment:
Thanks! My thought process was like since CardContentHorizontal extends CardContent and doesn't have any additional properties or styles of its own, using the TemplateSpec from CardContent makes sense here.
May be I am wrong.

Reply:
that's correct, but in that case we should be omitting the namespace entirely instead of declaring an empty TemplateSpec

Reply:
But since CardContentHorizontalLarge extends CardContentHorizontal, having an empty namespace will throw an error, right?

Reply:
nope, it would just reference the parent's interfaces